src/ui/AnotherQuickSwitcherModal.ts

```diff
@@ -90,6 +90,41 @@ export class AnotherQuickSwitcherModal
     this.scope.register(["Mod"], "K", () => {
       document.dispatchEvent(new KeyboardEvent("keydown", { key: "ArrowUp" }));
     });
+    // Delete whole input line
+    this.scope.register(["Mod"], "D", () => {
+      const elem = this.inputEl;
+      const str = elem.value;
+      elem.setSelectionRange(0, str.length);
+      elem.setRangeText("");
+    });
+    // Delete word at cursor or before cursor if at end of string
+    this.scope.register(["Mod"], "W", () => {
```

Comment:
>I think it is not necessary because we can delete a word by OS shortcut keys (ex: In Windows, `Ctrl+Backspace` ).

Yes you are right! Thanks for pointing this out, this is good to know! I also found some other shortcuts that works here: https://en.wikipedia.org/wiki/Table_of_keyboard_shortcuts#Text_editing

I am testing this on Linux and I can use:
- CTRL + BACKSPACE : Delete word to the left of cursor
- CTRL + DELETE : Delete word to the right of cursor
- CTRL + LEFT : Move cursor to the beginning of the current word or the previous word
- CTRL + RIGHT : Move cursor to end of the current word or the next word

Very good to know! 

There is a small difference between this PR's delete word and the OS alternatives, but I agree it would be better to stick with OS alternatives to avoid confusion. I have deleted that part of the PR.